src/views/managers/managers.js

```diff
@@ -78,24 +78,58 @@ const selectRow = {
   },
 };
 
-const Managers = () => {
+// transforms data from API into a format that can be used for bootstrap-table-next
+const convertManagersDataForTable = (managersArray) => {
+  const convertedManagers = managersArray.map(manager => {
+    manager.fullName = `${manager.firstName} ${manager.lastName}`;
+    
+    // #####################################################
+    // not exactly sure if this is how we're hoping to implement displaying the "status" field?
+    // I took my best guess with it for now :)
+    if (manager.lastActive || !manager.archived) {
```

Comment:
I think this is fine for now, but I will follow up with @htharker42 about what the statuses on the mockups actually mean. I also see "Awaiting Approval" but don't know how that differs from "Pending" nor how they get into those states